r-package/R/dictionary.R

```diff
@@ -0,0 +1,57 @@
+#'
+#'
+#' Open the data set's dictionary.
+#'
+#' @param dataset a string containing the name of a basedosdados' data set, the dictionary of which you are asking for.
+#' @param billing_project_id a string containing your billing project id. If you've run `set_billing_id` then feel free to leave this empty.
+#'
+#' @return Returns a dictionary of the data set, allowing the user to check factors' meanings.
+#'
+#'
+#'
+#' @examples
+#'
+#' \dontrun{
+#'
+#'
+#' dictionary("br_inep_censo_escolar")
+#'
+#' dictionary("br_me_rais")
+#'
+#' }
+#'
+#'
+#' @import bigrquery
+#' @import glue
+#' @importFrom rlang abort is_character
+#' @importFrom magrittr %>%
+#' @export
+#'
+#'
+
+dictionary <- function(
+  dataset,
+  billing_project_id = get_billing_id(),
+  page_size = 100000) {
+
+  if(billing_project_id == FALSE) {
+
+    rlang::abort("You haven't set a Project Billing Id. Use the function `set_billing_id` to do so.")
+
+  }
+
+  if(!rlang::is_string(query)) {
```

Comment:
o objeto `query` ainda não foi criado aqui e gera o erro
`Error in rlang::is_string(query) : object 'query' not found`

Para teste, ignorei esse if e rodou certinho!

Reply:
eu sou burro vei, era pra ter mudado esse `query` pra `dataset` hahaha, obrigado querido